registration/serializers.py

```diff
@@ -109,6 +116,60 @@ def create(self, validated_data):
 
         return family
 
+    def update(self, instance, validated_data):
+        students_data = validated_data.pop("students")
+        students = (
+            [instance.parent]
+            + list(instance.children.all())
+            + list(instance.guests.all())
+        )
+
+        instance.email = validated_data.get("email", instance.email)
+        instance.home_number = validated_data.get("home_number", instance.home_number)
+        instance.cell_number = validated_data.get("cell_number", instance.cell_number)
+        instance.work_number = validated_data.get("work_number", instance.work_number)
+        instance.preferred_number = validated_data.get(
+            "preferred_number", instance.preferred_number
+        )
+        instance.address = validated_data.get("address", instance.address)
+        instance.preferred_comms = validated_data.get(
+            "preferred_comms", instance.preferred_comms
+        )
+        instance.notes = validated_data.get("notes", instance.notes)
```

Comment:
we can remove these and do `return super().update(instance, validated_data)` at the very end! the `super()` will run the default update on the family object, which will do all this for us